superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/controlPanel.tsx

```diff
@@ -447,4 +447,12 @@ const config: ControlPanelConfig = {
   ],
 };
 
+if (isFeatureEnabled(FeatureFlag.GENERIC_CHART_AXES)) {
```

Comment:
Can we just use ternary operator when declaring config? Like
```
isFeatureEnabled(FeatureFlag.GENERIC_CHART_AXES)) ? 
  { label: t('Shared query fields'),
    expanded: true,
    controlSetRows: [[xAxisControl]],
  } : sections.legacyTimeseriesTime
```

Reply:
Sorry, I meant to write
```
isFeatureEnabled(FeatureFlag.GENERIC_CHART_AXES)) ? 
  { label: t('Shared query fields'),
    expanded: true,
    controlSetRows: [[xAxisControl]],
  } : null
```
🤦

Reply:
oh right, I forgot about that pattern 👍

Reply:
Actually this currently only applies to controls, not control sections. But I added the `null` option to sections, too (tested that it works exactly like for controls): https://github.com/apache/superset/pull/20097/commits/60e7ba08c3f2df8f240ca9649cbac1ccfd3a9f4c